README.markdown

```diff
@@ -81,16 +81,18 @@ which should output 'nvm' if the installation was successful. Please note that `
 
 For manual install create a folder somewhere in your filesystem with the `nvm.sh` file inside it. I put mine in `~/.nvm`.
 
-Or if you have `git` installed, then just clone it, and check out the latest version:
+Or if you have `git` installed:
 
-```sh
-git clone https://github.com/creationix/nvm.git ~/.nvm && cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`
-```
-
-To activate nvm, you need to source it from your shell:
+1. clone this repo
+2. check out the latest version
+3. activate nvm by sourcing it from your shell
 
 ```sh
-. ~/.nvm/nvm.sh
+(
+  git clone https://github.com/creationix/nvm.git ~/.nvm
+  cd ~/.nvm
+  git checkout `git describe --abbrev=0 --tags --match "v[0-9]*" origin`
```

Comment:
Sadly it's not documented - I'm just very cautious about adding new `git` arguments. That seems old enough that it won't be a concern.